app/src/main/java/com/xinto/opencord/ui/screen/Login.kt

```diff
@@ -102,14 +105,13 @@ fun LoginScreen(
     }
 
     if (viewModel.showCaptcha) {
-        HCaptcha
-            .getClient(context)
-            .verifyWithHCaptcha(viewModel.captchaSiteKey)
+        hCaptcha.verifyWithHCaptcha()
             .addOnSuccessListener {
                 viewModel.login(it.tokenResult)
             }
             .addOnFailureListener {
-                //TODO
+                //TODO do more than just showing a toast i guess
+                Toast.makeText(context, "Captcha failed, try again", Toast.LENGTH_LONG).show()
```

Comment:
Shouldn't you retry keep retrying here?

Reply:
I kinda don't know how to implement that so let's just leave it like this for now

Reply:
i suppose you can just trigger it again with .verifyWithHCaptcha

Reply:
That seems a bit recursive, it might look bad in compose too